src/classes/UTIL_Currency.cls

```diff
@@ -35,6 +35,8 @@
 */
 public without sharing class UTIL_Currency implements UTIL_Currency.Interface_x {
 
+    public static final String CURRENCY_ISO_CODE_FIELD = 'CurrencyIsoCode';
```

Comment:
Instead of using text value throughout code, it is recommended to use a Constant. If there is a text value that is a misspelled, and there is no unit test, the error will be generated in run time. Using a constant removes run-time errors.

Reply:
Understood, noting that there are dozens of uses of `CurrencyIsoCode` as s string throughout the code base already.